plugins/github-deployments/src/components/GithubDeploymentsCard.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { LinearProgress } from '@material-ui/core';
+import { InfoCard, MissingAnnotationEmptyState, useApi } from '@backstage/core';
+import { useAsync } from 'react-use';
+import { githubDeploymentsApiRef } from '../api';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import {
+  GITHUB_PROJECT_SLUG_ANNOTATION,
+  isGithubDeploymentsAvailable,
+} from '../Router';
+import GithubDeploymentsTable from './GithubDeploymentsTable/GithubDeploymentsTable';
+
+const GithubDeploymentsComponent = ({
+  projectSlug,
+  last,
+}: {
+  projectSlug: string;
+  last: number;
+}) => {
+  const api = useApi(githubDeploymentsApiRef);
+  const [owner, repo] = projectSlug.split('/');
+
+  const { loading, value, error } = useAsync(
+    async () => await api.listDeployments({ owner, repo, last }),
+  );
+
+  if (loading) {
+    return (
+      <InfoCard title="GitHub Deployments">
+        <LinearProgress />
+      </InfoCard>
+    );
+  }
+  if (error) {
+    return (
+      <InfoCard title="GitHub Deployments">
+        Error occured while fetching Data
```

Comment:
no info at all about why? :) Maybe `<ResponseErrorPanel error={error} />` could be helpful (also from `core`)